src/components/spinner/Spinner.scss

```diff
@@ -0,0 +1,34 @@
+body {
+	background: var(--color-bg-main);
+	display: flex;
+	align-items: center;
+}
+
+.spinner-wrapper {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	gap: 10px;
+}
+
+.spinner {
+	margin: 0 auto;
+	width: 70px;
+	height: 70px;
+	border: 4px solid var(--color-bg-active);
+	border-radius: 50%;
+	border-top-color: var(--color-primary);
+	animation: spin 1s linear infinite;
+}
+
+.spinner-small {
+	margin: 0;
+	width: 20px;
+	height: 20px;
+}
+
+@keyframes spin {
```

Comment:
Replace @keyframes to the top. It is a good practice to place @keyframes animations at the top of your CSS file, before any other styles or rules, to ensure that they are defined and loaded first by the browser.